### PR TITLE
More than 2 Cores

### PR DESCRIPTION
**Pull request by Baltmann01**
I added the recipe for using more than 2 cores in installation, example, tests or vignettes.
Parts are based on the suggestion of @aitap in Discussion #61. 
I hope I implemented their suggestions correctly, as I'm not familiar with the issue too much.

### CODE REVIEW

File: website/code_issues.qmd [hunk 1]
```diff
@@ -383,4 +383,44 @@ CRAN doesn't allow negative `warn` options. This setting will turn off all warni
...
+
+## Problem
+
+You are using more than 2 CPU cores in your examples, vignettes, tests or during installation. 
+
+## Solution
+
+Make sure that you set the maximum number of cores used to 2 in examples, vignettes or tests. 
+During installation, many modern build systems use the `-j` flag. 
+<!-- This was recommended by 'aitap' in a discussion on GitHub -->
```
Comment (jasdumas) [inline website/code_issues.qmd:398]: Can you also add @aitap to the contributors section of the [preface.qmd](https://github.com/r-devel/cran-cookbook/blob/main/preface.qmd#L37)?

File: website/code_issues.qmd [hunk 2]
```diff
@@ -383,4 +383,44 @@ CRAN doesn't allow negative `warn` options. This setting will turn off all warni
...
+### Details
+
+::: {.callout-note title="CRAN Review Communication" appearance="simple" icon=false collapse=true}
+
+Please ensure that you do not use more than 2 cores in your examples, vignettes, etc.
+
+:::
+
+
+CRAN checks run on big, multi-core computers, running many `R CMD check` processes in parallel. Every process is allowed to use a maximum of two CPU cores to exercise parallel code in examples and unit tests. When using more than 2 cores, your package will get a NOTE from CRAN checks, saying that it took more CPU time than elapsed time.
```
Comment (jasdumas) [inline website/code_issues.qmd:409]: For consistency can you update the 'two CPU cores' to '2 CPU cores' to match the rest of the recipe?

File: website/code_issues.qmd [hunk 3]
```diff
@@ -383,4 +383,44 @@ CRAN doesn't allow negative `warn` options. This setting will turn off all warni
...
+
+Check: whether package can be installed, Result: NOTE\
+&emsp; Installation took CPU time 6.9 times elapsed time\
+\
+Check: examples, Result: NOTE\
+&emsp; Examples with CPU time > 2.5 times elapsed time
+  
+:::
+
+The automatic tests will only produce a NOTE when more than 2 cores are used in examples, vignettes, tests or the installation. The possibility to set the amount of cores should be given in any function and the default is ideally set to less than 2 cores.
```
Comment (jasdumas) [inline website/code_issues.qmd:421]: Nit wording update: 

```suggestion
Automatic tests will generate a NOTE if more than 2 cores are used in examples, vignettes, tests, or during installation. It’s best to provide an option for setting the number of cores in any function, with the default ideally set to fewer than 2 cores.
```

**Review by jasdumas — APPROVED** [review]

File: website/code_issues.qmd (hunk 1 above)
Comment (Baltmann01) [reply website/code_issues.qmd:398]: Sure! I will do that 😄

File: website/preface.qmd [hunk 4]
```diff
@@ -38,7 +38,8 @@ The CRAN Cookbook has been developed as part of a special project grant from the
 
 _Thanks to all contributors!_
 
-Lluís Revilla ([@llrs](https://github.com/llrs))
+Lluís Revilla ([@llrs](https://github.com/llrs))\
+[@aitap](https://github.com/aitap)
```
Comment (aitap) [inline website/preface.qmd:42]: Could you please add my name to the list? Sorry for not making it more obvious. 
```suggestion
Ivan Krylov ([@aitap](https://github.com/aitap))
```

**Pull request merged**